src/chainstate/stacks/miner.rs

```diff
@@ -745,8 +746,8 @@ pub mod test {
                     (miner_participant_principal, Value::Principal(PrincipalData::Standard(StandardPrincipalData::from(addr.clone()))))])
                 .expect("FATAL: failed to construct miner principal key"));
 
-            let miner_status = clarity_tx.connection().with_clarity_db_readonly(|ref mut db| {
-                let miner_status_opt = db.fetch_entry(&miner_contract_id, BOOT_CODE_MINER_REWARDS_MAP, &miner_principal)?;
+            let miner_status = clarity_tx.with_clarity_db_readonly(|db| {
+                let miner_status_opt = db.fetch_entry(&miner_contract_id, BOOT_CODE_MINER_REWARDS_MAP, &miner_principal).unwrap();
```

Comment:
Can we replace the `unwrap()` with an `expect()` and an error message?

Reply:
I can just revert this back to a `?`

Reply:
Never mind -- will change to expect -- the changes in clarity.rs made `with_clarity_db_readonly` return whatever the return type of the closure is (rather than previously a Result<T, ClarityError>), so the unwrap needed to move from 786 to here.